### DISCUSSION TITLE
Accounts + Proxies

### DISCUSSION THREAD

**Discussion by wiktorelka** [3urobeat/steam-comment-service-bot#247] [Q&A]
Hello, if i am using 20 accounts with 20 proxies, could i change the comment delay to 5 seconds or less?

**Comment by 3urobeat** [comment]
Hi, depends on how many comments you request at once for a profile.  

From the perspective of your accounts, yes, you wouldn't even need a commentdelay at all when every account is on another IP.

However, a profile can only get so many comments in a short timeframe and your comments will probably fail at some point when requesting a lot of comments for one profile.  
I would try setting the commentdelay to 2500 when requesting around 10-20 comments at once.  
Try it out and if comments fail you can always increase the value again :)
